### PR TITLE
AkkaHttpServer: prevent access to not yet initialized fields

### PR DESCRIPTION
**Pull request by jrudolph**
Refs #10107

### CODE REVIEW

File: transport/server/play-akka-http-server/src/main/scala/play/core/server/AkkaHttpServer.scala [hunk 1]
```diff
@@ -167,6 +167,45 @@ class AkkaHttpServer(context: AkkaHttpServer.Context) extends Server {
...
+      ReloadCacheValues(
+        resultUtils = serverResultUtils,
+        modelConversion = modelConversion,
+        serverDebugInfo = reloadDebugInfo(tryApp, provider)
+      )
+    }
+  }
+
+  // ----------------------------------------------------------------------
+  // CAUTION
```
Comment (raboof) [inline transport/server/play-akka-http-server/src/main/scala/play/core/server/AkkaHttpServer.scala:203]: :+1:

**Comment by raboof** [comment]
needs a scalafmt

**Comment by jrudolph** [comment]
We should try to backport, can that be done automatically?

**Comment by mkurz** [comment]
@Mergifyio backport 2.8.x

**Comment by mkurz** [comment]
Let's see

**Comment by jrudolph** [comment]
Thanks ;)

**Pull request merged**